frameworks/helloworld/tests/test_sanity.py

```diff
@@ -32,12 +32,14 @@ def configure_package(configure_security):
     try:
         sdk_install.uninstall(FOLDERED_SERVICE_NAME, package_name=PACKAGE_NAME)
 
+        service_config = {"service": {"name": FOLDERED_SERVICE_NAME, "user": "root"}}
```

Comment:
The path forward is:

1. Do this for now
2. Make sure we publish a Hello World with 1.8 as its min version that has nobody as default (make a ticket!)
3. Stop doing this once that version is released.